tests/dat/actions/sdk/swift4/hello.swift

```diff
@@ -0,0 +1,7 @@
+func main(args: [String:Any]) -> [String:Any] {
+    if let name = args["name"] as? String {
+        return [ "greeting" : "Hello \(name)!" ]
+    } else {
+        return [ "greeting" : "Hello stranger!" ]
+    }
+}
```

Comment:
Since some of Swift 3 and Swift 4 tests are the same, should we use a base directory to store common files so we don't keep duplicates?  Maybe I'm being too picky?